scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -785,6 +795,9 @@ private void processAlertRecord(final Record record, final Connection connection
 					errorHandler.onError(alert.getPeer(), alert.getLevel(), alert.getDescription());
```

Comment:
I talked about : [`ErrorHandler`](https://github.com/eclipse/californium/blob/master/scandium-core/src/main/java/org/eclipse/californium/scandium/ErrorHandler.java)
It was introduce long time ago to get error feedback from DTLS Layer, but it seems to me that what you introduce (`MessageCallback.onError`+`SessionListener.handshakeFailed`) do the job better. So we could remove it now ?